components/observations/ObservationUrlMapping.tsx

```diff
@@ -0,0 +1,29 @@
+import {AvalancheCenterID, AvalancheCenterWebsites} from 'types/nationalAvalancheCenter';
+
+export const GenerateObservationShareLink = (centerId: AvalancheCenterID, observationId: string) => {
+  let url: string | undefined = AvalancheCenterWebsites[centerId];
```

Comment:
`Record<AvalancheCenterID, string>` ensures at compile time that every center ID is in there, so I don't think this is truly `string | undefined` right?